src/osm-gps-map/osm-gps-map-osd-classic.c

```diff
@@ -23,6 +23,7 @@
 #include <stdlib.h>  // abs
 #include <string.h>
 #include <math.h>    // M_PI/cos()
+#include <stdio.h>
```

Comment:
This include is required to avoid the warning

```
cc1: warning: command line option '-fvisibility-inlines-hidden' is valid for C++/ObjC++ but not for C [enabled by default]
../maep-qt/src/osm-gps-map/osm-gps-map-osd-classic.c: In function 'osd_render_hr':
../maep-qt/src/osm-gps-map/osm-gps-map-osd-classic.c:1622:9: warning: implicit declaration of function 'snprintf' [-Wimplicit-function-declaration]
         snprintf(str, 4, "%03u", priv->hr.rate);
```